### PR TITLE
Rephrase references to the entire/other structured suffixes.

### PR DESCRIPTION
**Pull request by msporny**
This PR attempts to address feedback received from @mnot regarding how we refer to the components of a structured suffix. It introduces the "entire structured suffix" terminology, which notes that it can be composed of "other structured suffixes".

### CODE REVIEW

**Review by rhiaro — APPROVED** [review]

File: index.xml [hunk 1]
```diff
@@ -93,18 +93,28 @@ and only when, they appear in all capitals, as shown here.
...
-names containing a structured suffix are built from right to left (not left
-to right). Characters on the left-most side of the left-most "+" in a subtype
-name specify the base subtype name. Characters to the right of each "+" in
-a subtype name denote additional structured syntax suffixes.
+Media types MAY be registered with more than one structured suffix appended to
+the base subtype name. Characters on the left-most side of the left-most "+" in
+a subtype name specify the base subtype name. The entire structured suffix is
+all of the characters to the right of the first "+" sign in the media type,
+including the initial "+" sign itself. The entire structured suffix MAY be
+composed of one or more other structured suffixes.
```
Comment (OR13) [inline index.xml]: this is an improvement, but repeating with a neutral example would improve things, imo.

```
application/foo+bar+baz
```

`application` is the type.
`foo` is the subtype.
`+bar+baz` is the structured suffix.

File: index.xml [hunk 2]
```diff
@@ -93,18 +93,28 @@ and only when, they appear in all capitals, as shown here.
...
+is expected to process either the entire media type, or "+baz", or "+bar+baz".
+The processor is never expected to process "+bar" alone when presented with a
+"application/foo+bar+baz" media type, as that would be considered interpreting
+multiple structured suffixes out of order.
+    </t>
+    <t>
+Media types containing more than one structured suffix MUST be registered
+according to the procedure defined in [RFC6838]. A new base subtype name MUST
+only be registered with structured suffixes that are already registered in their
+own right in the
```
Comment (OR13) [inline index.xml]: I recognize you didn't write this, but its not great text imo.

I would like to see an example provided, for example:

```
application/foo+bar+baz
```

can only be registered if `+bar+baz` is already registered... is that what this is saying?

File: index.xml (hunk 1 above)
Comment (msporny) [reply index.xml]: Agreed, I'll add an example in the next pass.

File: index.xml (hunk 2 above)
Comment (msporny) [reply index.xml]: Yes, that's what this text is attempting to say. I'll provide an example in the next pass and try clarifying the language further.

Comment (msporny) [reply index.xml]: Fixed in 89fbd25974ead982d2093d681db0cb3cbad5f5ad.

File: index.xml (hunk 1 above)
Comment (msporny) [reply index.xml]: Fixed in 6573643fe64b591ba5856f5c909aed9026c2f147.

File: index.xml [hunk 3]
```diff
@@ -93,21 +93,33 @@ and only when, they appear in all capitals, as shown here.
...
-subtype name. The suffixes MUST be interpreted as ordered. Valid media type
-names containing a structured suffix are built from right to left (not left
-to right). Characters on the left-most side of the left-most "+" in a subtype
-name specify the base subtype name. Characters to the right of each "+" in
-a subtype name denote additional structured syntax suffixes.
+Media types MAY be registered with more than one structured suffix appended to
+the base subtype name. Characters on the left-most side of the left-most "+" in
+a subtype name specify the base subtype name. The entire structured suffix is
+all of the characters to the right of the first "+" sign in the media type,
+including the initial "+" sign itself. The entire structured suffix MAY be
```
Comment (TallTed) [inline index.xml:100]: Consistency throughout is vital to comprehension. I can make this into a PR, if that's better for you.
```suggestion
all of the characters to the right of the left-most "+" sign in the media type,
including the left-most "+" sign itself. The entire structured suffix MAY be
```

File: index.xml [hunk 4]
```diff
@@ -93,21 +93,33 @@ and only when, they appear in all capitals, as shown here.
...
+is expected to process either the entire media type, or "+baz", or "+bar+baz".
+The processor is never expected to process "+bar" alone when presented with a
+"application/foo+bar+baz" media type, as that would be considered interpreting
+multiple structured suffixes out of order.
     </t>
     <t>
+Media types containing more than one structured suffix MUST be registered
+according to the procedure defined in [RFC6838]. A new media type that utilizes
+a structured suffix MUST only be registered if the entire structured suffix
+is already registered in the
```
Comment (TallTed) [inline index.xml:121]: Sometimes, the negative phrasing is clearer.
```suggestion
a structured suffix MUST NOT be registered if the entire structured suffix
is not already registered in the
```

**Pull request merged**